script.js

```diff
@@ -0,0 +1,9 @@
+const inputs = document.querySelectorAll('.controls input');
+
+function handleUpdate() {
+  const suffix = this.dataset.sizing || '';
+  document.documentElement.style.setProperty(`--${this.name}`, this.value + suffix);
+}
+
+inputs.forEach(input => input.addEventListener('change', handleUpdate));
+inputs.forEach(input => input.addEventListener('mousemove', handleUpdate));
```

Comment:
Check your app on the mobile phone: the view is getting changed NOT when you move input, but only when you mouseUp. Try to fix it.

Reply:
DONE!